pkg/common.go

```diff
@@ -33,20 +33,45 @@ func DetailToString(d *checker.CheckDetail, logLevel log.Level) string {
 	}
 
 	var sb strings.Builder
-	sb.WriteString(fmt.Sprintf("%s: %s", typeToString(d.Type), d.Msg.Text))
+	// Non-structured results.
+	//nolint: nestif
+	if d.Msg.Finding == nil {
```

Comment:
This is hard to maintain with this many nesting. Can this be broken into separate funcs?

Reply:
done, thanks